kerastuner/tuners/bayesian.py

```diff
@@ -107,21 +112,19 @@ def _populate_space(self, trial_id):
             return {'status': trial_lib.TrialStatus.RUNNING,
                     'values': values}
 
-        def _upper_confidence_bound(x):
-            x = x.reshape(1, -1)
-            mu, sigma = self.gpr.predict(x, return_std=True)
-            return mu - self.beta * sigma
-
         optimal_val = float('inf')
         optimal_x = None
         num_restarts = 50
+        y_max = self.get_best_trials()[0].score
         bounds = self._get_hp_bounds()
         x_seeds = self._random_state.uniform(bounds[:, 0], bounds[:, 1],
                                              size=(num_restarts, bounds.shape[0]))
+        ac = utils.UtilityFunction(kind=self.acq, beta=self.beta).utility
```

Comment:
Prefer more descriptive name, like ```self.acquisition_function```. Also the logic to select the acquisition function should happen in ```__init__```